manifests/mc2.pp

```diff
@@ -0,0 +1,19 @@
+# == Class: seed_stack::mc2
+#
+# Runs a Mission Control container in Marathon.
+#
+class seed_stack::mc2(
+  $infr_domain,
+  $hub_domain,
+  $marathon_host = 'marathon.service.consul',
+) {
+  file { '/etc/marathon-apps': ensure => 'directory' }
+  ->
+  file { '/etc/marathon-apps/mc2.marathon.json':
```

Comment:
I'm wondering if `puppet-xylem` should actually handle the placement of these files... but it's mostly semantics. Something like:
```puppet
xylem::marathon_sync::group { 'mc2':
  content => template('seed_stack/mc2.marathon.json.erb')
}
```

I donno, just a thought..